### PR TITLE
ajudst ogr time type to json format

### PR DESCRIPTION
**Pull request by michaelmattig**
- [ ] I added an entry to [`CHANGELOG.md`](CHANGELOG.md) if knowledge of this change could be valuable to users.

---

Here is a brief summary of what I did:

<TEXT>

### CODE REVIEW

**Comment by michaelmattig** [comment]
[UI für OGR Datasets](https://trello.com/c/xBzHbJxU/1728-ui-f%C3%BCr-ogr-datasets)

File: services/src/api/model/datatypes.rs [hunk 1]
```diff
@@ -1719,13 +1720,122 @@ impl From<RasterPropertiesEntryType> for geoengine_datatypes::raster::RasterProp
...
     has_time: bool,
 }
 
+enum FormatStrLoopState {
+    Normal,
+    Percent(String),
+}
+
+impl DateTimeParseFormat {
+    pub fn custom(fmt: String) -> Self {
```
Comment (ChristianBeilschmidt) [inline services/src/api/model/datatypes.rs:1736]: This looks rather complicated. Maybe test it?

File: services/src/api/handlers/datasets.rs [hunk 2]
```diff
@@ -1407,6 +1413,63 @@ pub async fn delete_dataset_handler<C: ApplicationContext>(
...
+    ),
+    params(
+        ("volume_name" = VolumeName, description = "Volume name"),
+        ("file_name" = String, description = "File name")
+    ),
+    security(
+        ("session_token" = [])
+    )
+)]
+pub async fn list_volume_file_layers_handler<C: ApplicationContext>(
```
Comment (ChristianBeilschmidt) [inline services/src/api/handlers/datasets.rs:1438]: Add one success case test?

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: services/src/api/handlers/datasets.rs (hunk 2 above)
Comment (michaelmattig) [reply services/src/api/handlers/datasets.rs:1438]: c2418df70afee617ce159f7f6dcca966efacbbb8

File: services/src/api/model/datatypes.rs (hunk 1 above)
Comment (ChristianBeilschmidt) [reply services/src/api/model/datatypes.rs:1736]: Yeah, back then it was tested and then I copied it doesn't sound good.
Rather copy the test or do the conversion.

Comment (michaelmattig) [reply services/src/api/model/datatypes.rs]: 5b5c0e77ea24da8f4adb41af2be807332aa341ef

**Review by ChristianBeilschmidt — APPROVED** [review]

**Pull request merged**